packages/telegram-stamper/src/index.ts

```diff
@@ -0,0 +1,185 @@
+/// <reference lib="dom" />
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TelegramStamperError } from "./errors";
+
+declare global {
+  interface Window {
+    Telegram: any;
+  }
+}
+
+// config when telegram stamper is passed in an api key
+export type TTelegramStamperConfig = {
+  apiPublicKey: string;
+  apiPrivateKey: string;
+  cloudStorageKeySuffix?: string;
+};
+
+type CloudStorageAPIKey = {
+  apiPublicKey: string;
+  apiPrivateKey: string;
+};
+
+// Constant for default key name, note this should NOT be used directly, use the getCloudStorageKeyName() function as it includes the user passed key suffix
+const TURNKEY_CLOUD_STORAGE_KEY = "TURNKEY_API_KEY";
```

Comment:
I don't think you need this comment because this constant isn't exported, and so not usable by default. If we go with the custom key route (and forgo the prefix), this whole prefix logic goes away so the comment can go as well!

Reply:
The comment was more for developers who would be writing code for the Telegram stamper, but tbh youre right having a prefix just feels clunky, giving dev's full control is much better